src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,33 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
```

Comment:
don't use `magic numbers` in your code 


it means that ALL literals (etc. numbers) should be initialized as global constants 

f.e.:
const HEALTH_DEFAULT = 100;